applications/msai.md

```diff
@@ -0,0 +1,300 @@
+For a simpler initial alpha we could integrate IPFS as a pod making it simpler than the full Pod api [4].
+Distributed data could be verified as authentic by having a secure hash stored on the chain with the data in the separate pod.
+With trust being established by the blockchain layer, any blockchain node that provides faulty data as verified through smart contract hashes will be kicked out and forfiet their staking in the parachains native token.
+We also propose specific data format for anonymous tags such that no user identified private data would be exposed
+during the whole process. Additional layers of encryption and anonymity would be provided in subsequent stages.
+
+The initial user of this parachain would be our own app where users store any medical documents with automated information extraction, along with authenticated GPDR opt out.
+Sample screen of users view of how the data will be presented in our implementation can be seen here https://docs.google.com/presentation/d/1XEQ6qtTLwkmc6JgtpytEUJqhh1behskOWh3s4uE1cXw/edit?usp=sharing
+
+Such a platform would allow game theory incentives and smart contract based marketplaces for data trading to encourage adoption of the reference application. 
+
+Our team has years of research and development on medical data along with senior blockchain developers working with IT capable PhD accredited medical doctor(s). Data collection extraction and real time scaling is also part of Dr S Marvasti's and Dr Mahdi Ghandi's resume .
+Furthermore, Dr Marvasti has multiple years experience working with blockchain technologies including ZKP both for cross border fintech and for data analysis through AWS Neptune and AWS Aurora API based databases.
+This is a phased plan with part of the design to be implemented in this first phase, which each phase being usable .
+in summary benefits of blockchain for this data includes
+* Decentralised: Data ownership is not lost to any central authority and access is controlled through secure ownership of NFTs
+* Incentive to participate: various mechanisms  could be developed as smart tokens or native token of the parachain linked to DOT.
+* Users:  publishers of health can receive some amount of token as rewards from any sponsor on the platform. 
+* Public: The api and parachain would be public so that any other apps such that deal with medical data can use this parachain. 
+* The detailed API and openAPI specifications for GRPC and or REST will be developed as part of this proposal with reference to the Solid pod api as details depends on the public blockchain that the system is implemented in.
+### Project Details
+
+Data models
+
+We propose to store data encrypted at client side using the Pod api . The data would be anonymous at the storage layer and encrypted with access to ID available only to owner(s) of the NFT controlled via a smart contract. 
```

Comment:
Can you explain how ownership of an NFT could enable someone to decrypt data?

Reply:
I added some explanation it is by mechanism of the added logic to the offchain worker storage api that is specialised for this application.